Source/WebCore/css/StyleProperties.cpp

```diff
@@ -67,6 +67,11 @@ static bool isCSSWideValueKeyword(StringView value)
     return value == "initial"_s || value == "inherit"_s || value == "unset"_s || value == "revert"_s || value == "revert-layer"_s;
 }
 
+static bool isCSSWideValueIDKeyword(CSSValueID valueID)
+{
+    return valueID == CSSValueInitial || valueID == CSSValueInherit || valueID == CSSValueUnset || valueID == CSSValueRevert || valueID == CSSValueRevertLayer;
+}
+
```

Comment:
This function already exists here: https://searchfox.org/wubkat/rev/cd711e64b46522100059a32e3025f7875e4861c0/Source/WebCore/css/parser/CSSParserIdioms.h#63-75

Can we re-use that instead?

Or if we can't, I would rather that we check `value->isCSSWideKeyword()` instead (against the CSSValue, not the ID). We shouldn't introduce yet another function that does this (there's already quite a few, isUniversalKeyword, isCSSWideKeyword, etc.)